.travis/run.sh

```diff
@@ -9,22 +9,14 @@ if [[ "$(uname -s)" == "Darwin" ]]; then
     PATH="$PYENV_ROOT/bin:$PATH"
     eval "$(pyenv init -)"
 
-    if [[ "${OPENSSL}" != "0.9.8" ]]; then
-        # set our flags to use homebrew openssl
-        export ARCHFLAGS="-arch x86_64"
-        # if the build is static we need different LDFLAGS
-        if [[ "${CRYPTOGRAPHY_OSX_NO_LINK_FLAGS}" == "1" ]]; then
-            export LDFLAGS="/usr/local/opt/openssl/lib/libssl.a /usr/local/opt/openssl/lib/libcrypto.a"
-        else
-            export LDFLAGS="-L/usr/local/opt/openssl/lib"
-        fi
-        export CFLAGS="-I/usr/local/opt/openssl/include"
-        # The Travis OS X jobs are run for two versions
-        # of OpenSSL, but we only need to run the
-        # CommonCrypto backend tests once. Exclude
-        # CommonCrypto when we test against brew OpenSSL
-        export TOX_FLAGS="--backend=openssl"
+    # set our flags to use homebrew openssl
+    # if the build is static we need different LDFLAGS
+    if [[ "${CRYPTOGRAPHY_OSX_NO_LINK_FLAGS}" == "1" ]]; then
+        export LDFLAGS="/usr/local/opt/openssl/lib/libssl.a /usr/local/opt/openssl/lib/libcrypto.a"
+    else
+        export LDFLAGS="-L/usr/local/opt/openssl/lib"
     fi
+    export CFLAGS="-I/usr/local/opt/openssl/include"
```

Comment:
ARCHFLAGS because we never needed it and it was cargo culted from the dark days of me not knowing what I was doing. TOX_FLAGS because we need to run the commoncrypto tests and previously we selectively switched them off unless we were testing system openssl. This was done to improve test runtime and not duplicate effort, but now we have no 0.9.8 builders so we need to run commoncrypto everywhere. I think it'd be sufficient to create a new COMMONCRYPTO variable and run the commoncrypto tests only when that's set, but I'm not sure everyone agrees with me :grinning: